dipy/io/image.py

```diff
@@ -27,16 +27,6 @@ def save_nifti(fname, data, affine, hdr=None):
     result_img.to_filename(fname)
 
 
-def load_affine_matrix(fname):
-    """
-        Parameters
-        ---------
-        fname : str
-            The file containing the saved affine matrix.
-    """
-    return np.loadtxt(fname)
-
-
 def save_affine_matrix(fname, affine):
```

Comment:
While we're here, could you please get rid of `save_affine_matrix`? That looks like a wrapper around `np.savetxt`.

Reply:
ok, done in 4306c36